src/MobileBookingHeader/MobileBookingDetail.js

```diff
@@ -110,6 +119,34 @@ class MobileBookingDetail extends React.Component<Props, State> {
     expanded: false,
   };
 
+  componentDidMount() {
+    this.updateFAQSection();
+  }
+
+  componentDidUpdate() {
+    this.updateFAQSection();
+  }
+
+  updateFAQSection = () => {
+    const { booking } = this.props;
+    const { isPastBooking } = booking;
```

Comment:
hmm this is basically exactly the same as @jalbertsr's implementation, no? If so, maybe this should not be copy-pasted?

Reply:
I mean the entire method + the other methods, not the specific line :-)

Reply:
yes I know is an ugly copy-paste, I was thinking to create a folder `help/booking` with the methods that are used by both, but since half of the `MobileBookingDetail` and related files are also a copy paste maybe we should think on somehow merge these two components into one and create them smart enough to know if mobile or desktop  should be rendered.

Reply:
what do you prupose? I think that since this is a bit "urgent" we can do later a `refactor` issue.

Reply:
Good idea! Maybe for now it'd be enough to extract the functionality of these methods to 3 separate functions and instead of copying the entire source code then we could just call those functions.

Reply:
We can refactor everything else later.